features/step-definitions/task.3.js

```diff
@@ -0,0 +1,33 @@
+const { Given, When, Then } = require("@wdio/cucumber-framework");
+const { expect, $ } = require("@wdio/globals");
+const path = require("path");
+const DownloadPage = require("../pageobjects/download.page");
+const filename = "text.txt";
+const fs = require("fs");
+const file_path = path.join(path.join(process.cwd(), "tempDownload"));
+
+async function assertFileIsDownloaded(filename) {
+  const downloadedFilePath = path.join(
+    `${file_path}/${browser.capabilities.browserName}/${filename}`
+  );
+
+  const fileExists = await fs.existsSync(downloadedFilePath);
+
+  expect(fileExists).toBe(true);
+}
+
+When(/^I click download files$/, async () => {
+  DownloadPage.open();
```

Comment:
You should click on a File Download link on a home page. The page shouldn't open itself